src/schemas/events.ts

```diff
@@ -197,6 +197,12 @@ const zEventBase = ({ image }: SchemaContext) =>
         .optional(),
       subPages: z.array(reference("eventsSubPages")).optional(),
       sponsoringLevels: z.array(z.string()).optional(),
+      eventUpdatesChannels: z
+        .object({
+          newsletter: z.string().url().optional(),
+          whatsapp: z.string().optional(),
+        })
+        .optional(),
```

Comment:
```suggestion
      updatesChannels: z.array(z.object({
      	type: z.enum(['whatsapp', 'newsletter']),
      	link: z.object({
      		title: z.string(),
      		href: z.string().url(),
      	}),
      })).optional(),
```

This way, we can add more `type` easily and the `link` object is more expendable.